docs/quickstart.md

```diff
@@ -236,33 +236,26 @@ client.get_offline_features(observation_settings=settings,
 ## Step 6: Materialize feature value into offline/online storage
 While Feathr can compute the feature value from the feature definition on-the-fly at request time, it can also pre-compute 
 and materialize the feature value to offline and/or online storage. <br/>
-The `deploy` command line call will materialize feature value according to the feature generation config under feature_gen_conf.
 
-```bash
-feathr deploy
-```
-Or in Python:
 ```python
-job_res = client.materialize_features()
-```
-The following feature generation config is used to materialize feature value to Redis:
 
-```
-operational: {
-  name: generateWithDefaultParams
-  endTime: 2021-01-02
-  endTimeFormat: "yyyy-MM-dd"
-  resolution: DAILY
-  output:[
-  {
-      name: REDIS
-      params: {
-        table_name: "nycTaxiDemoFeature"
-      }
-   }
-  ]
-}
-features: [f_location_avg_fare, f_location_max_fare]
+from datetime import datetime, timedelta
+from feathr.client import FeathrClient
+from feathr.materialization_settings import (BackfillTime,
+                                             MaterializationSettings)
+from feathr.sink import RedisSink
+
+client = FeathrClient()
+# Backfill time range is [start, end].
+backfill_time = BackfillTime(start=datetime(2020, 5, 20), end=datetime(2020, 5,20), step=timedelta(days=1))
+redisSink = RedisSink(table_name="nycTaxiDemoFeature")
+# Materialize two features into a redis table.
+settings = MaterializationSettings("nycTaxiMaterializationJob",
+                                   backfill_time,
+                                   sinks=[redisSink],
+                                   feature_names=["f_location_avg_fare", "f_location_max_fare"])
+client.materialize_features(settings)
```

Comment:
Given this part is similar with the update in  "feature-generation.md", how about extract it into a code snippet markdown file and refer to them in two place? It can help to guarantee consistence in the future change.

Reply:
I'm not sure how to do that but if @Yuqing-cat you have some ideas maybe you can have a small PR doing this? We have quite a few places like this

Reply:
Seems can't find a easy way to achieve so, will make more investigation and resolve this for now.